### ISSUE TITLE
200 points is a very high minimum

### ISSUE THREAD

**Issue by paulirish** [ichinaski/hnnotifications#2]
Only ~14% of the current frontpage has over 200 points, but I'm interested in front-page items.

Can you lower the threshold to something like 20?

**Comment by ichinaski** [comment]
Good point. The presence of keywords doesn't require such a high score threshold any more. I'll sort this out soon. 

I think a possible solution would be to keep the minimum score restriction only when no keywords are involved in the criteria, to prevent the user from being constantly spammed. Getting a notification from any top item matching your keywords, regardless of the score (if you desire so) is quite reasonable. What do you think?
